app/views/ConfirmationPageView.scala.html

```diff
@@ -71,7 +71,11 @@
         content = Html(messages(s"cf.historic.document.request.confirmation.body-text2.${fileRole.name}"))
     )
 
-    @p(link(linkMessage = messages(s"cf.historic.document.request.confirmation.${fileRole.name}.link-text"),
+    <div class="govuk-!-margin-bottom-9">
```

Comment:
Could you please add the required style classes as part of link component (like below) instead of creating a new div? Also write test cases to check the presence of this style class as its important for this page layout.

  @p(link(linkMessage = messages(s"cf.historic.document.request.confirmation.${fileRole.name}.link-text"),
            location = returnLink,
            pClass = "govuk-body govuk-!-margin-bottom-9")
    )